extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -223,10 +223,10 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 				await this.bookTocManager.recovery();
 				vscode.window.showErrorMessage(loc.editBookError(updateBook.book.contentPath, e instanceof Error ? e.message : e));
 			} finally {
-				this.fireBookRefresh(targetBook);
```

Comment:
Would suggest updating the name - it's doing more than "firing a book refresh". I think `initialBookContents` is clearer and more accurate (the firing of the event isn't really that important to what the function is actually doing)